models/maisi_ct_generative/configs/inference.json

```diff
@@ -34,8 +34,8 @@
     "spatial_dims": 3,
     "image_channels": 1,
     "latent_channels": 4,
-    "output_size_xy": 512,
```

Comment:
Hi @borisfom , could you revert this size change? The default values are set by MAISI author